pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -257,3 +272,55 @@ func (c *cephStatusChecker) conditionMessageReason(condition cephv1.ConditionTyp
 
 	return condition, reason, message
 }
+
+// forceDeleteStuckPodsOnNotReadyNodes lists all the nodes that are in NotReady state and
+// gets all the pods on the failed node and force delete the pods stuck in terminating state.
+func forceDeleteStuckPodsOnNotReadyNodes(context *clusterd.Context, namespace string, clusterName string) error {
+	nodes, err := k8sutil.GetNotReadyKubernetesNodes(context.Clientset)
+	if err != nil {
+		return errors.Wrapf(err, "failed to force delete stuck pods.")
+	}
+	for _, node := range nodes {
+		pods, err := getPodsOnNode(context, node.Name, namespace, clusterName)
+		if err != nil {
+			logger.Errorf("Failed to get get pods on NotReady node %q. %v", node.Name, err)
+		}
+		for _, pod := range pods {
+			if err := k8sutil.ForceDeletePodIfStuck(context, pod); err != nil {
+				return errors.Wrapf(err, "skipping forced delete of stuck pod %q.", pod.Name)
```

Comment:
I doubt this error message is accurate.

Reply:
Instead of returning an error, just logging an error message.